lib/dradis/plugins/mapping_service.rb

```diff
@@ -0,0 +1,99 @@
+module Dradis
+  module Plugins
+    class MappingService
+      attr_accessor :integration, :rtp_id, :source
+
+      def initialize(args = {})
+        @integration   = args.fetch(:integration)
+        @rtp_id        = args.fetch(:rtp_id, nil)
```

Comment:
Updated the parameter to be called `destination` instead of `rtp_id` and I'm now passing in the full destination instead of parsing the `rtp id` it in the service